snippets/properties.json

```diff
@@ -0,0 +1,13 @@
+{
+	"Add datasource properties": {
+		"prefix": "qds",
+		"body": [
+			"quarkus.datasource.url=${1:jdbc:mariadb://localhost:3306/mydb}",
+			"quarkus.datasource.driver=${2:org.mariadb.jdbc.Driver}",
+			"quarkus.datasource.username=${3:developer}",
+			"quarkus.datasource.password=${4:developer}",
+			"quarkus.hibernate-orm.database.generation=${5:update}"
```

Comment:
Use choice to provide the list `none, create, drop-and-create, drop, update`

Reply:
excellent idea, this works well